.github/workflows/server_compatibility.yaml

```diff
@@ -10,7 +10,7 @@ on:
       hz_version:
```

Comment:
I think only this file's organization name is left. By the way, we can omit adding to some files if it is not reasonable to add for some files.

Reply:
For server compatibility workflow, directly released client versions are used. I think we don't need organization_name parameter for this